CHARTER.md

```diff
@@ -1,59 +1,61 @@
+Each member of the ART is eligible for voting if they have been a member for at least 30 days. For any proposal that impacts the whole project or processes, **approval requires a minimum of four votes with 75% approval among those voting**.
 
-If the majority cannot be reached, the proposal is considered rejected and can be brought to the ART again 1 month after the rejection at the earliest. If the same decision has been rejected twice, it cannot be brought to the ART again.
+Any vote must be open for at least 7 calendar days to account for different time zones and other obligations and should last no longer than 30 days. The voting period should be stated in the proposal in calendar days. The voting period begins on the day the pull request is opened for review and ends at midnight [AoE](https://en.wikipedia.org/wiki/Anywhere_on_Earth) on the last day of the voting period. Proposals can be extended for no longer than 30 days total. Extensions can be made before the end of the current voting period by the original submitter, their direct representative, or one of the ART chairs by updating the PR with a note stating the number of calendar days for the extension. **If approval is not reached within the voting period deadline, the proposal is rejected.**
 
-Any voting period must last at least 7 calendar days to account for different time zones and other obligations and should last no longer than 30 days. The voting period should be stated in the proposal. Proposals can be extended no more than twice and for no longer than 30 days total. Extensions can be made by the original submitter, their direct representative, or one of the ART chairs by updating the PR with a note stating the number of calendar days for the extension.
+A rejected proposal may only be submitted to the ART again after a miniumum of 30 days following a rejection. If the same proposal is rejected twice, it will not be considered again by the ART.
 
-ART members cast their votes by starting a review that either approves or rejects the decision suggested via a pull request. A reject vote is recorded by using the *Request changes* function of the PR review. Once the voting period is over or all ART members have voted, the chairs record the decision, objections, and other relevant information in the [`art-decisions` folder](art-decisions). Silence equals abstaining from a vote.
+ART members cast their votes by starting a review that either approves or rejects the proposal pull request. A rejection vote is recorded by using the *Request changes* function of the pull request review and providing a reason for their rejection in the comment. Once the voting period is over or all ART members have voted, the chairs record the decision, objections, and other relevant information in the [`art-decisions` folder](art-decisions).
```

Comment:
How about "by reviewing the pull request"? I don't know ... "starting a review" somewhat echoes GitHub terminology, but could also easily be taken to imply starting ones own PR; and at best it seems a bit over-burdened.

I also noticed at one point that we have the history folder of "proposals" and the *document* of accepted decisions ... although using git makes this a bit more awkward, I wonder if we should consider moving the approvals to a decisions folder, where the whole document could provide more detail than the summary in the decisions document (e.g., with a link), and to keep the namespace organized?

Reply:
I commented on another dimension of this below; however, from a process perspective, since all this stuff ultimately lands in a Git repo, currently under GitHub administration, we need a workflow which ends with the files and respective modifications being merged into `main` and landing in the right places in the repo.  So, it would be best if we didn't have to make many changes after the PR is approved.